protos/sources/vega/checkpoint/v1/checkpoint.proto

```diff
@@ -224,3 +226,45 @@ message AssetAction {
   bool erc20_bridge_stopped = 11;
   bool erc20_bridge_resumed = 12;
 }
+
+message ELSShare {
+  // Party ID of the liquidity provider.
+  string party_id = 1;
+  // The equity-like share of the liquidity provider, representing their share in the total provided liquidity expressed as a value between 0 and 1.
+  string share = 2;
+  // Amount committed. This is the total commitment made. It does not necessarily match the LP balance.
+  string supplied_stake = 3;
+  // LP stake, weighted according to the time the stake was committed and market growth.
+  string virtual_stake = 4;
+  // Average entry valuation. As time increases the weight of stake, this is the overall valuation of the current stake.
+  string avg = 5;
+}
+
+message MarketState {
+  // Unique ID for the market
+  string id = 1;
+
+  // The current equity-like share data for a market.
+  repeated ELSShare shares = 2;
+
+  // Current balance of the insurance pool.
+  string insurance_balance = 3;
+
+  // Total trade value which last determined the growth rate of the market.
+  string last_trade_value = 4;
+
+  // Total trade volume which last determined the growth rate of the market.
+  string last_trade_volume = 5;
+
+  // Succession window in nanonseconds: the timestamp when this market cannot be succeeded anymore.
+  int64 succession_window = 6;
+
+  // The underlying market should be present in case the successor market is enacted during the succession window.
+  // This field can be omitted if the market is not yet settled.
+  vega.Market market = 7;
```

Comment:
If it can be omitted should we mark it as optional?

Reply:
It's a checkpoint type, it's not something that should ever be exposed, and given that the proposals are now to be handled differently, this is likely to be removed anyway